pkg/ssh/ssh.go

```diff
@@ -371,17 +371,17 @@ func (l *SSHTunnelList) removeAndReAdd(e sshTunnelEntry) {
 	}
 	l.tunnelsLock.Unlock()
 	if err := e.Tunnel.Close(); err != nil {
-		klog.Infof("Failed to close removed tunnel: %v", err)
+		klog.InfoS("Failed to close removed tunnel", "err", err)
 	}
 	go l.createAndAddTunnel(e.Address)
 }
 
 func (l *SSHTunnelList) Dial(ctx context.Context, net, addr string) (net.Conn, error) {
 	start := time.Now()
 	id := mathrand.Int63() // So you can match begins/ends in the log.
-	klog.Infof("[%x: %v] Dialing...", id, addr)
+	klog.InfoS("Dialing...", "id", id, "addr", addr)
 	defer func() {
-		klog.Infof("[%x: %v] Dialed in %v.", id, addr, time.Since(start))
+		klog.InfoS("Dialed in.", "id", id, "addr", addr, "timeElapsed", time.Since(start))
```

Comment:
Using `addr` and `address` in this PR. These should be consistent.  The https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging#example-of-migrating-klog-call suggests via example that it would be addr.

Reply:
I second making the name consistent, but instead using `addr` we should use `address`. Sorry for confusion with naming on the original KEP it was written much earlier then the current migration guide https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md. Please consult it for the up to date status.

Reply:
awesome I was looking for that doc with those keywords :100: